servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/completable/CompletableToPublisherTest.java

```diff
@@ -40,7 +40,8 @@
 
 class CompletableToPublisherTest {
     @RegisterExtension
-    final ExecutorExtension<Executor> executorExtension = ExecutorExtension.withCachedExecutor();
+    static final ExecutorExtension<Executor> executorExtension = ExecutorExtension.withCachedExecutor()
```

Comment:
Likely requires changing the name to `EXEC` or `EXECUTOR_EXTENSION` to satisfy CheckStyle. Even it does not the convention should be applied.

Reply:
Decided for changing it all to EXEC, to make it shorter